src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -64,6 +67,7 @@ define(function (require, exports, module) {
         this._color = tinycolor(color);
         this._originalColor = color;
         this._redoColor = null;
+        this._isUpperCase = PreferencesManager.get("uppercaseColors");
```

Comment:
I think the error is coming in here. Notice how `this._color` is also in the prototype, but this is not. I have gone back and forth including it there too or not, but I've always hit issues either way. I have a feeling that somehow, the unit tests editor is not picking up the updated `this._isUpperCase` value, and including it in the prototype as well will fix it. My question then is how to ensure listening to the `change` will properly update the prototype and not the object.

Yea, the inline color editor is rather a mess...